File: frontend/src/features/comment/types/comment.ts
export interface Comment {
  id: number;
  boardId: number;
  memberId: number;
  parentId: number | null;
  content: string;
  writer: string;
  deleted: boolean;
  createdAt: string;
  updatedAt: string;
}

export interface CommentCreateRequest {
  content: string;
  parentId?: number | null;
}

export interface CommentUpdateRequest {
  content: string;
}

File: backend/src/main/java/com/board/backend/comment/domain/Comment.java
package com.board.backend.comment.domain;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class Comment {

    private Long id;
    private Long boardId;
    private Long memberId;
    private Long parentId;
    private String content;
    private String writer;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private LocalDateTime deletedAt;
}
